cpp/BUILD_MANUAL.md

```diff
@@ -7,7 +7,7 @@ The purpose of  this manual is to provide you with some information on how to co
 ### Build Tools 
 
 [CMake](https://cmake.org/) is our main build tool for the C++ version of this Kata. 
-In addition to CMake, we are using [vcpkg](https://github.com/microsoft/vcpkg) to avoid compiling our third-party libraries.  
+In addition to CMake, we are using [CPM](https://github.com/TheLartians/CPM.cmake) CMake dependency Management.  
```

Comment:
We don't need to mention CPM in the build tools I think, since it is a pure CMake process that does not a priori have any requirements for people cloning the repo.